data/cg-teams.ts

```diff
@@ -325,49 +383,74 @@ export default class TeamGenerator {
 		teamStats: TeamStats,
 		species: Species,
 		movesSoFar: Move[],
-		ability: string
+		ability: string,
+		level: number,
 	): number {
 		if (!move.exists) return 0;
 		// this is NOT doubles, so there will be no adjacent ally
 		if (move.target === 'adjacentAlly') return 0;
 
+		const adjustedStats: StatsTable = {
```

Comment:
Is this based on an in-game stats calculation? It's not immediately clear why you scale Atk and SpA by (level/100)^2.